about_us.md

```diff
@@ -28,11 +28,11 @@ We develop Kubernetes Open Source Operators using [Operator Lifecycle Manager (O
 
 ## What we plan to do?
 
-We plan to improve the exiting operators by adding Master (control plane) fencing capabilites to SNR and NHC, which currently **only** work for worker nodes.
+We plan to improve the existing operators by adding Master (control plane) fencing capabilities to SNR and NHC, which currently **only** work for worker nodes.
 Furthermore, we have few work in progress operators:
 
 - [Machine Deletion Remediation](/remediation/machine-deletion/machine-deletion-remediation) - Remediates nodes by deleting the associated OpenShift machine. Triggered by NHC.
-- [HA-SNO](https://github.com/medik8s/ha-sno) - High Availability based on two Single Node Openshift .
+- [HA-SNO](https://github.com/medik8s/ha-sno) - High Availability based on two Single Node OpenShift.
```

Comment:
I don't think it was wrong. IIRC It is optional to write `Openshift` or `OpenShift` but not `openshift`

Reply:
okay, I can revert this

Reply:
I don't think there is a choice. Looking the docs it's OpenShift.